src/test/msixtest/main.cpp

```diff
@@ -17,9 +17,11 @@ int main( int argc, char* argv[] )
     #else
     auto find = exePath.find_last_of('/');
     #endif
+    const char* path = nullptr;
     if (find != std::string::npos)
     {
         exePath = exePath.substr(0, find);
+        path = exePath.c_str();
     }
-    return msixtest_main(argc, argv, exePath.c_str());
+    return msixtest_main(argc, argv, path);

```

Comment:
>path [](start = 37, length = 4)

Why the change here?

Reply:
I forgot to add this to the description. This is the bug that if you run the tests from the location where the exe is located, the test will fail to find the location of the test data. 

This is because instead of passing the folder location of the exe, it will pass the exe itself, so it will try to find
(eg. msixtest.exe\testdata\...) which clearly doesn't exists. If we pass nullptr, then the TestPath object will not add the root.


---
In reply to: [302265031](https://github.com/Microsoft/msix-packaging/pull/170#discussion_r302265031) [](ancestors = 302265031)

Reply:
I see the difference now.

---
In reply to: [302272977](https://github.com/Microsoft/msix-packaging/pull/170#discussion_r302272977) [](ancestors = 302272977,302265031)